src/zeek.bif

```diff
@@ -4071,26 +4071,29 @@ MMDB_lookup_result_s MMDB::Lookup(const struct sockaddr* const sa)
 bool MMDB::StaleDB()
 	{
 	struct stat buf;
-	using Clock = std::chrono::steady_clock;
-	std::chrono::time_point<Clock> now = Clock::now();
 
 	if ( lookup_error )
 		return true;
 
-	// Only perform stat once per 5 minutes.
-	using Min = std::chrono::minutes;
-	if ( std::chrono::duration_cast<Min>(now - last_check).count() < 5 )
+
+	static double mmdb_stale_check_interval = zeek::id::find_val("mmdb_stale_check_interval")->AsInterval();
```

Comment:
The `static` here looks like a microoptimization to me which introduces subtleties. Are you sure it is needed and we cannot just get a fresh value each time?

Reply:
This might be called often enough and we do that elsewhere. Will keep it in.

https://github.com/zeek/zeek/blob/254d7ebfa2f39c26bfd18c79f7b6f525eb713fc6/src/zeek.bif#L5094

Reply:
Yes. In common configurations a mmdb lookup might be done 4 times for each connection.